assets/javascripts/wizard/initializers/custom-wizard-field.js.es6

```diff
@@ -15,6 +15,10 @@ export default {
     FieldComponent.reopen({
       classNameBindings: ["field.id"],
 
+      textType: function() {
```

Comment:
@fzngagan We don't use this old style of computed property. I know that the other code in this file does, but we should use the new approach when writing new code.